app/src/main/java/org/fossasia/pslab/activity/MainActivity.java

```diff
@@ -271,6 +271,14 @@ public void onBackPressed() {
             drawer.closeDrawers();
             return;
         }
+
+        if( fragment instanceof HomeFragment){
```

Comment:
Can you please do a code reformat to all the files? There are plenty of spaces missing and extra gaps.